app/Domain/Clients/Repositories/Clients.php

```diff
@@ -136,8 +140,9 @@ public function isClient($values): bool
 			name = :name AND street = :street LIMIT 1';
 
             $stmn = $this->db->database->prepare($sql);
-            $stmn->bindValue(':name', $values['name'], PDO::PARAM_STR);
-            $stmn->bindValue(':street', $values['street'], PDO::PARAM_STR);
+
+            $stmn->bindValue(':name', $values->name, PDO::PARAM_STR);
+            $stmn->bindValue(':street', $values->street, PDO::PARAM_STR);
```

Comment:
I know this was there before but this is actually weird. Why are we getting client by street? I think isClient() using the name only is fine. Can you check if we are actually using this method anywhere? If not, let's just remove it.